internal/splitmerge/splitmerge_test.go

```diff
@@ -34,9 +34,11 @@ func (b *BufferCloser) Close() error {
 	return nil
 }
 
-//               ┌─> copy data per 1 byte    ─>┐
+//	┌─> copy data per 1 byte    ─>┐
+//
 // data ─> split ├─> copy data per ... bytes ─>├─> merge
-//               └─> copy data per 42 bytes  ─>┘
+//
```

Comment:
this is `make fmt` artefact

Reply:
Lol. I found a workaround - add emoji:
```
// 🙈               ┌─> copy data per 1 byte    ─>┐
// 🙉 data ─> split ├─> copy data per ... bytes ─>├─> merge
// 🙊               └─> copy data per 42 bytes  ─>┘
```

PS: or cdots =)
```
// ·               ┌─> copy data per 1 byte    ─>┐
// · data ─> split ├─> copy data per ... bytes ─>├─> merge
// ·               └─> copy data per 42 bytes  ─>┘
```